distributor-node/src/app/index.ts

```diff
@@ -90,14 +90,11 @@ export class App {
     }
   }
 
-  private hideSecrets(config: Config) {
-    const displaySafeConfig: DisplaySafeConfig = {
-      ...config,
-      keys: config.keys?.map((k) => _.mapValues(k, () => '###SECRET###' as const)),
-      operatorApi: _.mapValues(config.operatorApi, () => '###SECRET###' as const),
-    }
-
-    return displaySafeConfig
+  private hideSecrets(config: Config): DisplaySafeConfig {
```

Comment:
I had to rework this functionality slightly since hiding value on a nested path was quite hard to do (and very unreadable). I'm open to alternatives if there are issues with the new approach

Reply:
Since this new approach already sacrifaces the typesafety, as it's very difficult to maintain here, I decided to drop the `DisplaySafeConfig` type as well and replace it with a simpler `Record<string, unknown>` type.

I also reworked your `hideConfigPath` function to work recursively, as I think that makes it both clearer and more futureproof.

Here is my PR: https://github.com/kdembler/joystream/pull/1